### DISCUSSION TITLE
Calculation of the dynamic pressure at the tower

### DISCUSSION THREAD

**Discussion by kamalhabbal** [OpenFAST/openfast#1828] [Q&A] [answered]
Hello,

i'm recently trying to know at which specific position on the node of the tower the dynamic pressure (e.g. TwN1DynP) is calculated. Is it calculated at the front surface of the node? And is it possible to adjust the position of the calculation around the tower surface?

Thank you for your help in advance! 

Best regards,
Kamal

**Comment by jjonkman** [comment, marked as answer]
Dear @kamalhabbal,

The dynamic pressure can be calculated and output at each aerodynamic analysis node along the tower.  These nodes are located along the centerline of the (possible deflected) tower.  The dynamic pressure is computed simply as 1/2*AirDens*Vrel^2, where Vrel is the relative  speed between the ambient wind inflow (without rotor induction or tower influence) and tower velocities.

Please note the impact of the tower on the wind at the rotor can be calculated via the tower influence model (for upwind rotors) or the tower shadow model (for downwind rotors), but this distortion of the wind inflow, including the tower surface effects you are asking about, does not directly influence the calculation of the tower dynamic pressure output.

Best regards,

**Reply by kamalhabbal** [reply]
Dear @jjonkman,

Thank you for the clarification! 

Does the blade dynamic pressure calculation take into account the tower influence model? (Is the modified velocity vector used after calculating the tower effect?) I'm curious to know how much the dynamic pressure changes on the tower as well as the blade as it passes in front of the tower using OpenFAST.

Thank you again in advance.

Best regards,

**Reply by jjonkman** [reply]
Dear @kamalhabbal,

In AeroDyn, the tower influence and tower shadow models influence the wind local to rotor, and so, these will influence the dynamic pressure at blade analysis nodes.  But the rotor will not impact the dynamic pressure of the tower analysis nodes.

Best regards,

**Reply by kamalhabbal** [reply]
Dear @jjonkman,

Thank you for the clarification again! Have a nice day!

Best regards,
